pkg/crypto/tinkcrypto/primitive/jwe/ecdhes/ecdh_aead_private_key_manager.go

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
I don't fully understand the jwe reference in the path. JWE is not a crypto primitive. Can you clarify the meaning?

Reply:
This new package represents a new `special` primitive not be used directly by Crypto service. The aim is to create a special keys/crypto primitive for JWE operations and be able to store and exchange keys in localKMS. The best way to do this is to create new fully Tink supported keys and primitives for this special operation. 

The aim is for components requiring JWE envelopes to directly call these new primitives  in as many steps as possible. These steps would be:
1. get a key handle that supports JWE building
2. call Encrypt()

Instead  of a multiple steps process where Encrypt() would be broken down to the separate parts of the JWE message, ie get recipient public key, create a nonce, create a CEK, encrypt plaintext, extract tag, derive recipient's wrapping key, etc.

That's why `jwe` is in the path of this new special Tink primitive.

Reply:
JWE reference is now removed from this change. It will be introduced in a another PR specifically for JWE outside of crypt package.